### ISSUE TITLE
We could add link to "How is your website impacting the planet?" from Website Carbon Calculator

### ISSUE THREAD

**Issue by BogdanCerovac** [aardrian/DoesYourSiteDeserveRecognition.com#4]
Website carbon footprint (directly connected to page weight, technology behind it and hosting provider) should maybe also be added to the list.

I can make a PR if wanted...

**Comment by aardrian** [comment]
Good idea. Looking this site over, it needs a few other updates. I will add this and make one larger edit.

**Comment by aardrian** [comment]
For my later reference:
https://www.websitecarbon.com/website/doesmysitedeserverecognition-com/
